p/themes/Fresh-Nord/nord.css

```diff
@@ -0,0 +1,1228 @@
+@charset "UTF-8";
+
+:root {
+	/* Set sans-serif & mono fonts */
+	--sans-font: Inter, Lato,Helvetica,"IBM Plex Sans","Roboto",-apple-system,BlinkMacSystemFont,"Nimbus Sans L",Avenir,"Noto Sans", "Segoe UI",Arial,Helvetica,"Helvetica Neue",sans-serif;
+	--mono-font: "mononoki Nerd Font","IBM Plex Mono","Roboto Mono","Ubuntu Mono","Fira Code","Overpass Mono", Monaco,"Droid Sans Mono",monospace;
+	/*This is my dark themed color scheme*/
+	--bg: #242933;
+	--accent-bg: rgb(46, 52, 64);
+	--text: #eceff4;
+	--text-light: #d8dee9;
+	--border: #88c0d0;
+	--accent: #81a1c1;
+	--accent-light: #bf616a;
+	--code: #ebcb8b;
+	--code-bg: #2e3440;
+}
+
+
+/*=== GENERAL */
+/*============*/
+
+@font-face {
+	/*	src: local('Open Sans'), local('OpenSans'),
+		url('../fonts/OpenSans.woff2') format('woff2'),
+		url('../fonts/OpenSans.woff') format('woff');*/
+}
+
+html, body {
+	background: var(--bg);
+	color: var(--text);
+	font-family: var(--sans-font) !important;
+}
+
+/*=== Links */
+a {
+	background: none !important;
```

Comment:
is this `!important` really needed?

Reply:
> is this `!important` really needed?

Its not now, since I added a enabled a border to distinguish active buttons... I actually deleted the line completely, seems fine.

![image](https://user-images.githubusercontent.com/18448587/172953001-4a0ce0da-be87-4aff-a713-f8e93dce70d3.png)